src/types/parameters.ts

```diff
@@ -1,8 +1,8 @@
 import { Options } from './options';
 
 interface ParametersBase {
-	medium: Options['medium'];
-	warningLevel: Options['warningLevel'];
+	usermedium: Options['medium'];
+	warning: Options['warningLevel'] | 'no';
 }
```

Comment:
This appears to be changing the purpose of the parameters. These parameters were originally meant to reflect the friendly naming, not the naming used by the W3C service. The actual conversion of the names and values to the ones expected by the W3C service previously occurred closer to the actual request.

Reply:
More would need to be renamed if you did change the purpose, including `ParametersBase`, along with the filename itself, to match our conventions for consistency between export names and filenames.

But why is the purpose changing? Why is this not occurring closer to the actual request like it used to?

Reply:
The conversion isn't happening any further or closer than it was previously really. Previously the conversion from our options to the requests parameter names was happening inside of `buildRequestURL`, that was also the only place that used this type. Now what I have done instead is continue doing the conversion inside of what is now `buildRequestURLParams`, but for the `POST` request the conversion is being done directly inside of `validateText`, which is slightly closer to the request I suppose, but not meaningfully so.

Maybe what I can do differently is move the building of the parameters into the `retrieveValidation` function which will then build them based on what `method` has been passed to it.